### PR TITLE
feat: fix resourceNames

### LINKED ISSUE #23: Logging: Cannot specify resources when fetching log entries
In LoggingImpl, the ListLogEntriesRequest is built and only specifies the projectId as the resource names: https://github.com/googleapis/google-cloud-java/blob/64189021c41c0f96b8a96beefb737cbf76be22c8/google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java#L632-L653

The end-user should be able to somehow set the resource names fields.

[RPC Specification](https://cloud.google.com/logging/docs/reference/v2/rpc/google.logging.v2#listlogentriesrequest)

Note: this will require some substantial public interface changes.

### PR DESCRIPTION
**Pull request by pmakani**
Fixes #23

### CODE REVIEW

File: google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java [hunk 1]
```diff
@@ -177,6 +180,21 @@ public static EntryListOption sortOrder(SortingField field, SortingOrder order)
     public static EntryListOption filter(String filter) {
       return new EntryListOption(OptionType.FILTER, filter);
     }
+
+    /** Returns an option to specify a organization to the log entries to be listed. */
```
Comment (elharo) [inline google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java]: a organization to --> an organization for

File: google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java [hunk 2]
```diff
@@ -177,6 +180,21 @@ public static EntryListOption sortOrder(SortingField field, SortingOrder order)
...
+    public static EntryListOption organization(String organization) {
+      return new EntryListOption(OptionType.ORGANIZATION, organization);
+    }
+
+    /** Returns an option to specify a billingAccount to the log entries to be listed. */
```
Comment (elharo) [inline google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java]: to the --> for the

File: google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java [hunk 3]
```diff
@@ -177,6 +180,21 @@ public static EntryListOption sortOrder(SortingField field, SortingOrder order)
...
+    public static EntryListOption billingAccount(String billingAccount) {
+      return new EntryListOption(OptionType.BILLINGACCOUNT, billingAccount);
+    }
+
+    /** Returns an option to specify a folder to the log entries to be listed. */
```
Comment (elharo) [inline google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java]: to the --> for the

File: google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java (hunk 1 above)
Comment (pmakani) [reply google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java]: Done.

File: google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java (hunk 2 above)
Comment (pmakani) [reply google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java]: Done.

File: google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java (hunk 3 above)
Comment (pmakani) [reply google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java]: Done.

**Review by simonz130 — APPROVED** [review]

**Pull request merged**